content/docs/concepts/secrets.md

```diff
@@ -310,7 +304,11 @@ $ pulumi up
 Password: [secret]
 ```
 
-By default, configuration values are saved in plain text. To explicitly denote a plain text or unencrypted configuration value, pass the `--plaintext` flag. This flag can be used to indicate that you did not want an encrypted secret.
+### Explicitly denote unencrypted values
+
+By default, configuration values are saved in plain text. Given this, there may be times where you need to explictly denote a value as plain text or unencrypted. For example, for security purposes, the Pulumi CLI tries to detect when something that looks like an API token or password is supplied to as a Pulumi configuration value. There are some scenarios where it will incorrectly assume a string is a secret and will alert you in the command line while aborting the task.
+
+To circumnavigate this, you can pass the `--plaintext` flag when creating your configuration value. This flag can be used to indicate that you did not want an encrypted secret.
```

Comment:
```suggestion
To avoid this, you can pass the `--plaintext` flag when creating your configuration value. This flag can be used to indicate that you did not want an encrypted secret.
```